### ISSUE TITLE
Build workflow broken in github

### ISSUE THREAD

**Issue by chrisant996** [chrisant996/clink#81]
The "Build" workflow added in a26c65065db2f4c7d1f49c781cddf14bf955bea7 stopped working.  It looks like [ilammy/msvc-dev-cmd](https://github.com/ilammy/msvc-dev-cmd) was changed to detect some errors and abort.

That seems to be causing the following errors in the Build workflow that was added to the Clink repo:
```
Run ilammy/msvc-dev-cmd@v1
Found with vswhere: C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat
The system cannot find the path specified.
Error: [ERROR:winsdk.bat] Windows SDK 8.1 : 'include' not found
Error: [ERROR:VsDevCmd.bat] *** VsDevCmd.bat encountered errors. Environment may be incomplete and/or incorrect. ***
Error: [ERROR:VsDevCmd.bat] In an uninitialized command prompt, please 'set VSCMD_DEBUG=[value]' and then re-run
Error: [ERROR:VsDevCmd.bat] vsdevcmd.bat [args] for additional details.
Error: [ERROR:VsDevCmd.bat] Where [value] is:
Error: [ERROR:VsDevCmd.bat]    1 : basic debug logging
Error: [ERROR:VsDevCmd.bat]    2 : detailed debug logging
Error: [ERROR:VsDevCmd.bat]    3 : trace level logging. Redirection of output to a file when using this level is recommended.
Error: [ERROR:VsDevCmd.bat] Example: set VSCMD_DEBUG=3
Error: [ERROR:VsDevCmd.bat]          vsdevcmd.bat > vsdevcmd.trace.txt 2>&1
Error: Could not setup Developer Command Prompt: invalid parameters
```

I don't personally need this workflow (I use local workflows, including `premake5 release` which does more).  So for now I've disabled the Build workflow.  If it gets fixed, I'll re-enable it.  Or if it stays broken long enough, I may delete the workflow.

**Comment by chrisant996** [comment]
@koppor have you encountered this in the project(s) from which the workflow was copied?  Maybe a fix is known elsewhere?

**Comment by chrisant996** [comment]
Abandoning build workflow.  Wasn't adding value, and is too expensive to maintain.

**Comment by koppor** [comment]
> @koppor have you encountered this in the project(s) from which the workflow was copied? Maybe a fix is known elsewhere?

I crafted it for myself - trial and error for hours until I got #4 ready. Not following up from then.

> Abandoning build workflow. Wasn't adding value, and is too expensive to maintain.

My hope of https://github.com/chrisant996/clink/pull/4 that

- incoming PRs can automatically checked for trivial errors (build errors, ...)
- non-experienced users can get a build at each commit

Since both of these aspects is not needed anymore, I think, this can be closed.
